app/src/main/java/nl/tudelft/bbw/controller/CryptoController.java

```diff
@@ -58,4 +64,37 @@ public final String decryptString(String data) {
             throw new RuntimeException("Error - Could not decode string: " + e);
         }
     }
+
+    /**
+     * encryptFile method
```

Comment:
According to our SEM TA, consistency of our whole code style is really important.